source/includes/examples/_find-the-center.html.md

```diff
@@ -0,0 +1,182 @@
+# Find the Center Example
+
+In this example, we'll walk you through the various statements that are part of the Find the Center project, including the simulator file and the Inkling file. Find the Middle was created as a very basic example of Inkling and how to connect to a simulator.
+
+The general premise of Find the Center is that the computer (via Inkling) takes a step (move) of -1, 0, or 1, and gets rewarded for staying in the center (1) for each round. The max is 2 and the min is 0.
+
+## Inkling File
+
+###### Schema
+
+```inkling
+schema GameState
+    Int32 value
+end
+```
+
+The `GameState` schema has one record, `value`, assigned the constrained type of Int32.
+
+```inkling
+schema PlayerMove
+    Int32{-1, 0, 1} delta
+end
+```
+
+The `PlayerMove` schema has one record, `delta`, assigned the constrained type of Int32 to allow for negative and positive values.
```

Comment:
record -> field
Should Int32 get back ticks?

The `PlayerMove` schema has one field, `delta`, with type `Int32`. The `Int32` type is constrained to three possible values: -1, 0, and 1.